wordpress-mobile/WordPress-Android
# Create countries block

Fixes #8680 

This PR adds the Countries block to the DWMY stats. I'm reusing the old, javascript map which seems to work fine. We can replace it with a library if we feel like it or it doesn't suffice.

The landscape mode doesn't look great - there is an issue raised to fixed landscape over all stats - https://github.com/wordpress-mobile/WordPress-Android/issues/8785

To test:
* Open Stats
* Go to Day/Week/Month/Year tabs
* You see a Countries block at the bottom of the screen
* The countries block shows a clickable map of the world with the countries in the list highlighted
* There is a `View more` button available when there are 6+ countries
* The `View more` button links to the full list of Countries
